### PR TITLE
[vulkan] Try to enable Vulkan test in macOS presubmit

### PR DESCRIPTION
**Pull request by bobcao3**
Related issue = #

### CODE REVIEW

File: .github/workflows/presubmit.yml [hunk 1]
```diff
@@ -426,7 +426,7 @@ jobs:
           rm -rf $HOME/Library/Python/3.8/lib/python/site-packages/taichi
           .github/workflows/scripts/unix_build.sh
         env:
-          CI_SETUP_CMAKE_ARGS: -DTI_WITH_OPENGL:BOOL=OFF -DTI_WITH_CUDA:BOOL=OFF -DTI_WITH_CC:BOOL=OFF -DTI_WITH_VULKAN:BOOL=OFF -DTI_BUILD_TESTS:BOOL=ON
```
Comment (ailzhang) [inline .github/workflows/presubmit.yml:429]: Can we also run `ti diagnose` below to make sure vulkan backend is available?

**Comment by ailzhang** [comment]
@bobcao3 Looks like arch fallback indeed happened...

```
2021-11-11T20:22:06.1634030Z cc: False
2021-11-11T20:22:06.1634300Z cpu: True
2021-11-11T20:22:06.1634580Z metal: True
2021-11-11T20:22:06.1634860Z opengl: False
2021-11-11T20:22:06.1635160Z cuda: False
2021-11-11T20:22:06.1635440Z vulkan: False
```

**Comment by bobcao3** [comment]
> @bobcao3 Looks like arch fallback indeed happened...
> 
> ```
> 2021-11-11T20:22:06.1634030Z cc: False
> 2021-11-11T20:22:06.1634300Z cpu: True
> 2021-11-11T20:22:06.1634580Z metal: True
> 2021-11-11T20:22:06.1634860Z opengl: False
> 2021-11-11T20:22:06.1635160Z cuda: False
> 2021-11-11T20:22:06.1635440Z vulkan: False
> ```

I'm kind of confused as I have TI_WITH_VULKAN set to ON, but it seems it did not compile with it.

**Review by AmesingFlank — APPROVED** [review]

**Review by ailzhang — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**